sonar-db/src/main/resources/org/sonar/db/permission/PermissionMapper.xml

```diff
@@ -45,17 +45,20 @@
     u.login as login, u.name as name, u.email as email, lower(u.name), u.id
   </sql>
 
-  <select id="selectUsersByQuery" parameterType="map" resultType="org.sonar.db.permission.UserRef">
-    select distinct <include refid="userColumns" />
-    <include refid="usersByQuery"/>
-    order by lower(u.name), u.name, u.id
+  <select id="selectLoginsByPermissionQuery" parameterType="map" resultType="string">
+    select u.login
+    from (
+      select distinct <include refid="userColumns" />
```

Comment:
Last point, you should see with duke in order to :
- either validate that this is the right query
- either improve the query